### ISSUE TITLE
When we use ^1.4.0 it takes 1.5.0 and in IE 11 it shows forEach error in the library.

### ISSUE THREAD

**Issue by vinodhrajamuralidharan** [mitchlloyd/ember-islands#55]
- In package.json, if we use ^1.4.0 it takes 1.5.0 and in IE 11 it shows forEach error in the library. But 1.4.0 works fine.

**Comment by basemj** [comment]
https://github.com/mitchlloyd/ember-islands/commit/961ca4d77b645d1768cfbddf90bd44cfe5d5ba0a#diff-8b7471ed248975f5eb624ebde9297e01R15

**Comment by acorncom** [comment]
@vinodhrajamuralidharan would you able to send in a PR to fix it?

**Comment by rwjblue** [comment]
Fix would be something akin to:

```js
// from
document.querySelectorAll('[data-component]').forEach(function() {})

// to
let elements = document.querySelectorAll('[data-component]');
for (let i = 0; i < elements.length; i++) {
 // do stuff....
}
```

**Comment by mitchlloyd** [comment]
Does this seem like it would do the trick? https://github.com/mitchlloyd/ember-islands/pull/56

**Comment by mitchlloyd** [comment]
Published as ember-islands@1.5.1
